docs/low_level.md

```diff
@@ -0,0 +1,136 @@
+)
+
+print(model_response.parts[0].content)
+#> Paris
+print(usage_info)
+"""
+Usage(requests=0, request_tokens=56, response_tokens=1, total_tokens=57, details=None)
+"""
+```
+
+## Advanced Example with Tool Calling
+
+You can also use the low-level API to work with function/tool calling.
+
+Even here we can use Pydantic to generate the JSON schema for the tool:
+
+```python
+from pydantic import BaseModel
+from typing_extensions import Literal
+
+from pydantic_ai.low_level import model_request
+from pydantic_ai.messages import ModelRequest
+from pydantic_ai.models import ModelRequestParameters
+from pydantic_ai.tools import ToolDefinition
+
+
+class Divide(BaseModel):
+    """Divide two numbers."""
+
+    numerator: float
+    denominator: float
+    on_inf: Literal['error', 'infinity'] = 'infinity'
+
+
+async def main():
+    # Make a request to the model with tool access
+    model_response, cost = await model_request(
+        'openai:gpt-4.1-nano',
+        [ModelRequest.user_text_prompt('What is 123 / 456?')],
+        model_request_parameters=ModelRequestParameters(
+            function_tools=[
+                ToolDefinition(
+                    name=Divide.__name__.lower(),
+                    description=Divide.__doc__ or '',
+                    parameters_json_schema=Divide.model_json_schema(),
+                )
+            ],
+            allow_text_output=True,  # Allow model to either use tools or respond directly
+        ),
+    )
+    print(model_response)
+    """
+    ModelResponse(
+        parts=[
+            ToolCallPart(
+                tool_name='divide',
+                args={'numerator': '123', 'denominator': '456'},
+                tool_call_id='pyd_ai_2e0e396768a14fe482df90a29a78dc7b',
+                part_kind='tool-call',
+            )
+        ],
+        model_name='gpt-4.1-nano',
+        timestamp=datetime.datetime(...),
+        kind='response',
+    )
+    """
+    print(cost)
+    """
+    Usage(
+        requests=0, request_tokens=55, response_tokens=7, total_tokens=62, details=None
+    )
+    """
+```
+
+## When to Use Low-Level API vs Agent
+
+The low-level API is ideal when:
+
+1. You need more direct control over model interactions
+2. You want to implement custom behavior around model requests
+3. You're building your own abstractions on top of model interactions
+
+For most application use cases, the higher-level [`Agent`][pydantic_ai.Agent] API provides a more convenient interface with additional features such as built-in tool execution, structured output parsing, and more.
+
+## OpenTelemetry Instrumentation
+
+As with [agents][pydantic_ai.Agent] you can enable OpenTelemetry/logfire instrumentation with just a few extra lines
+
+```python {title="low_level_instrumented.py" hl_lines="1 6 7"}
+import logfire
+
+from pydantic_ai.low_level import model_request_sync
+from pydantic_ai.messages import ModelRequest
+
+logfire.configure()
+logfire.instrument_pydantic_ai()
```

Comment:
Other pydantic-ai docs like https://ai.pydantic.dev/logfire/ use `Agent.instrument_all()`, not `logfire.instrument_pydantic_ai()`.

Reply:
https://github.com/pydantic/pydantic-ai/pull/1625/

Reply:
does that not make it look like you have to use logfire and can't use plain otel?

Reply:
that's what others are recommending already, I'd be open to having a "usage without logfire section" which shows `Agent.instrument_all()`.

Reply:
what do you mean by others?

Reply:
sorry, long day, that's what I'm recommending everywhere else and seeing everywhere since it's consistent with how logfire can instrument other libraries.